packages/tools/client-debugger/client-debugger-view/src/index.ts

```diff
@@ -4,20 +4,20 @@
  */
 
 /**
- * Contains an extensible debug visualizer / editor for the Fluid client.
+ * This library contains a simple React-based visualizer for the Fluid Devtools.
  *
  * @remarks
  *
- * The entry-point to this package is {@link RootView}, a {@link https://react.dev/reference/react/Component | React Component}
+ * The entry-point to this package is {@link DevtoolsPanel}, a {@link https://react.dev/reference/react/Component | React Component}
  * for displaying debug information, which can be added to your Fluid-backed React app.
  *
  * @privateRemarks TODO: Add examples once the API surface has solidified.
  *
  * @packageDocumentation
  */
 
-export { AudienceMember } from "./Audience";
-export { AudienceMemberViewProps } from "./components";
 export { MessageRelayContext } from "./MessageRelayContext";
-export { RootView, RootViewProps } from "./RootView";
+export { RootView as DevtoolsPanel, RootViewProps as DevtoolsPanelProps } from "./RootView";
```

Comment:
I figured I would leave the internal naming alone for now, since we have a number of modules already whose names are confusingly similar. But for external consumers, I think this is more semantically meaningful.

Open to suggestions for better names though :)